.github/CONTRIBUTING.md

```diff
@@ -7,6 +7,14 @@
 Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing the bug.
 If that is the case, [open a new issue][new issue]. Make sure to include a clear title, description, and as much relevant information as possible.
 
+*   For feature requests, refer [here](../docs/template-featureRequest.md).
+*   For bug reports, refer [here](../docs/template-bugReport.md).
+*   For new contributor introduction, refer [here](../docs/template-contributorIntro.md).
+*   For help requests, refer  [here](../docs/template-helpRequest.md).
+*   For other work like refactoring, updating documentation, or performance improvement, simply provide a clear **issue title** and **description** of the work to be done.
```

Comment:
Two things here:
- Extraneous whitespace between `*` and `For` (all items) and between `refer` and `[here]` (item 4).
- Extraneous empty lines after this paragraph. One should be the max.